### ISSUE TITLE
dashboard logs + chunk analytics

### ISSUE THREAD

**Issue by crherman7** [crherman7/rechunk#70]
**Is your feature request related to a problem? Please describe.**
Currently we have no insights into what people are doing in the dashboard. Additionally we have analytics tied to how often analytics are read, etc.

**Describe the solution you'd like**
Create logs for the dashboard + analytics for the chunks.

**Describe alternatives you've considered**
Do nothing and keep everything anonymous.

**Comment by devs9** [comment]
@crherman7 🤔 can we implement this optionally?
in case some users would prefer to be anonymous and not send some analytics, but as an optional extra it would be great to have that option

**Comment by crherman7** [comment]
> @crherman7 🤔 can we implement this optionally? in case some users would prefer to be anonymous and not send some analytics, but as an optional extra it would be great to have that option

yeah fair point. I guess I mean more like light-weight analytics....# of chunks, # of reads of specific chunks - lightweight logs.....nothing super specific
